ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -5,47 +5,72 @@ set -o errexit
 set -o pipefail
 set -x
 
+# Runs the supplied command until the non-empty output file is created
+# or retries_max is reached.
+# Takes 2 arguments - the command/function and the output file
 run_scorecard() {
-        local retries_max=3
-        local attempt_num=1
-        until [[ $(jq . "$2") ]]
-        
-        do
-            "$1"
-            if (( attempt_num==retries_max ))
-            then
-                    echo "Retry attempt number: $attempt_num of $retries_max of scorecard tests failed. No more attempts. "
-                    return 1
-            else
-                    echo "Retry attempt number: $attempt_num of $retries_max failed. Retrying."
-                    sleep 5
-                    ((attempt_num++))
-            fi
-        done
+  local retries_max=3
+  local attempt_num=1
+  until [ -s "$2" ]
+
+  do
+    if (( attempt_num>retries_max )); then
+      echo "All $retries_max attempts of scorecard tests failed. No more attempts. "
+      return 1
+    else
+      echo "Attempt $attempt_num of $retries_max."
+      sleep 5
+      ((attempt_num++))
+    fi
+      "$1" "$2"
+  done
```

Comment:
Please do code formatting  (code style changes) in separate commit withing the same PR.

Current approach complicates review.